<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SNeicer's Doom Mod Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabw_MainTabs">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>0</y>
      <width>591</width>
      <height>561</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="btn_RefreshMods">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SourcePortDir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Source port .exe:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ActivateMod">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_UpSelected">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_DisableMod">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_BrowseModDir">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>490</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_ActiveMods">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>231</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineE_SourcePortDir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_BrowseSourcePortDir">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>430</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ModDir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mod dir:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineE_ModDir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_DownSelected">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_Mods">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Additional settings</string>
     </attribute>
     <widget class="QCheckBox" name="check_AVG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Austin Virtual Gaming (advancing to the next level after 20 minutes) ( -avg )</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_FastMonsters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fast monsters ( -fast )</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_NoMonsters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>No monsters ( -nomonsters )</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_RespawningMonsters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Respawning monsters ( -respawn )</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_NoAutoExec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable autoexec.cfg ( -noautoexec )</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gBox_CustomStartMap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Custom start map</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QTextEdit" name="tEdit_CustomStartMapName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_CustomMapName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Map (number for Doom 2 and eXmX for Doom 1):</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gBox_CustomDifficulty">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Custom Difficulty</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="cBox_CustomDifficulty">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>I'm too young to die (skill 1)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hey, not too rough (skill 2)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hurt me plenty (skill 3)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ultra-Violence (skill 4)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nightmare! (skill 5)</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_CustomDifficulty">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Difficulty:</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_CommandPrompt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>323</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Command Prompt (WIP and currently not working!)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tEdit_CustomCommandPrompt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>561</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_SaveAdditional">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save additional settings in preset</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_SaveCommandPrompt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>296</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save current launch command prompt instead of mods (WIP)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_Play">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_LoadPreset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SavePreset">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Preset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>566</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Preset</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cBox_ActivePreset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DeletePreset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddNewPreset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ProgInfo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>590</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SNeicer - 2022&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Created using python and PyQt5&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
